core/markup/src/SelectTool.ts

```diff
@@ -252,7 +252,7 @@ class MoveHandle extends ModifyHandle {
   }
   public modify(ev: BeButtonEvent): void {
     const evPt = MarkupApp.convertVpToVb(ev.viewPoint);
-    const dist = evPt.minus(this._lastPos!);
+    const dist = evPt.minus(this._lastPos ?? ({ x: 0, y: 0, z: 0 }));
```

Comment:
```suggestion
    assert(undefined !== this._lastPos);
    const dist = evPt.minus(this._lastPos;
```

The program logic is written to ensure that `_lastPos` is always initialized before `modify` is invoked. If that's not the case, we have made a logic error. Defaulting to (0,0,0) will produce surprising results and hide our mistake.

Reply:
updated